docs/Contributing.adoc

```diff
@@ -6,11 +6,38 @@ Herein lies the contribution guidelines for helping out with this project. Do ta
 
 === General rules
 
-* The Ansible <<coc,Code of Conduct>> still applies.
-* For git messages, branch names, ..., follow <<gitstyle,Git Style Guide>>
-* Should you wish to work on a completely new standard, GREAT, but please file an issue for tracking.
+* The Ansible link:https://docs.ansible.com/ansible/latest/community/code_of_conduct.html[Code of Conduct] still applies.
+* For git messages, branch names, ..., follow link:https://github.com/agis/git-style-guide[Git Style Guide].
```

Comment:
It seems like this is an individual's GitHub repository? Is there a more official git style document? If not, we should make our own copy rather than linking out.

Reply:
IIUC we can make our own copy, as long as we keep the license and copyright notice.

Regarding a more official guide, this one takes inspiration from https://www.kernel.org/doc/html/latest/process/submitting-patches.html which is less digest IMO.